connectors/src/admin/cli.ts

```diff
@@ -17,8 +18,15 @@ import { launchSlackSyncOneThreadWorkflow } from "@connectors/connectors/slack/t
 import { Connector } from "@connectors/lib/models";
 import { GoogleDriveFiles } from "@connectors/lib/models/google_drive";
 import { NotionDatabase, NotionPage } from "@connectors/lib/models/notion";
+import { SlackConfiguration } from "@connectors/lib/models/slack";
+import {
+  nango_client,
+  nangoDeleteConnection,
+} from "@connectors/lib/nango_client";
 import { Result } from "@connectors/lib/result";
 
+const { NANGO_SLACK_CONNECTOR_ID = "" } = process.env;
```

Comment:
I think we should error if it's undefined at the time we need it and not rely on a default.

Reply:
Done!